src/app/component/layout/components/search-popup/search-popup.component.html

```diff
@@ -54,7 +54,7 @@ <h3>Tips & Tricks</h3>
 
         <ng-template [ngIf]="(isNewsSearchFound === false && isTipsSearchFound === false)">
           <br/><br/>
```

Comment:
don't use `<br>` it's bad practice, because it's not semantic!

Reply:
fixed